WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -268,6 +271,20 @@ protected SendControlViewModel(WalletService walletService, string title)
 						return;
 					}
 
+					BitcoinAddress customChangeAddress = null;
+					if (!string.IsNullOrWhiteSpace(CustomChangeAddress))
```

Comment:
Add the condition to not use the change address if it is turned off in UIConfig. This is to avoid accidental usage of customChangeAddress with leftover data.

Reply:
We aren't doing that with the `CustomFee` either.

Reply:
We should then. There can be an unlikely combination of user action when this field still contains value but the feature is turned off.

Reply:
Done.